src/ui-kit/Search/Search.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Input } from 'antd'
+
+import './search.scss'
+
+import { SEARCH_DEFAULT_TEXT } from '../constants'
+
+const { Search } = Input
+
+export default () => {
```

Comment:
avoid this please, your components should have a name, now they are just anonymus functions:
```
const Search = () => </div>
export default Search
```